.github/workflows/publish-stage.yaml

```diff
@@ -7,6 +7,9 @@ on:
     tags-ignore:
       - "*"
 
+permissions:
+  id-token: write
```

Comment:
This is less scary than it sounds: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings